_tutorials/2016-05-10-iosxr-ansible-md.md

```diff
@@ -152,18 +151,50 @@ Success rate is 100 percent (5/5), round-trip min/avg/max = 1/5/20 ms
 RP/0/RP0/CPU0:ios#
 ```
 
-Let's copy public part of key from Ubuntu box and allow access without password. For IOS-XR image 6.1.x commands below:
 
-run vi /root/.ssh/authorized_keys
+**Doesn't work for me for now, looking for fix**
+Let's copy public part of key from Ubuntu box and allow access without password. For IOS-XRv image 6.1.x commands below:
 
 
-run sed -i.bak -e '/^PermitRootLogin/s/no/yes/' /etc/ssh/sshd_config_operns
+```
+run vi /vagrant/home/.ssh/authorized_keys
+```
 
-run service sshd_operns restart
+## Run the playbooks
 
-run chkconfig --add sshd_operns
+At Ubuntu box let's configure Ansible prerequisite. 
+We need to configure 2 files: ansible_hosts and ansible_env
+First file ansible_hosts contain ip address of our xr box. Also we will specify user for reaching out to machine: "ansible_ssh_user=vagrant"
```

Comment:
The first file: ansible_hosts contains the ip address of the XR instance.
We also specify a user to connect to the machine: "ansible_ssh_user=vagrant"